src/guide/code-owners.md

```diff
@@ -228,6 +227,7 @@ This list is intended to make it easier to identify which email list to include
 * `jdk.internal.vm.compiler` – Compiler (Removed in [22](https://bugs.openjdk.org/browse/JDK-8318027))
 * `jdk.internal.vm.compiler.management` – Compiler (Removed in [22](https://bugs.openjdk.org/browse/JDK-8318027))
 * `jdk.pack` – Tools (Removed in [14](https://bugs.openjdk.org/browse/JDK-8234596))
+* `jdk.random` – Core Libs (Unknown when it was Removed)
```

Comment:
Maybe add a reference to this RN?  https://bugs.openjdk.org/browse/JDK-8331843

Reply:
Yes. Forgot to add that. Good catch!